packages/react-router/index.ts

```diff
@@ -200,10 +200,13 @@ export type {
   ClientActionFunctionArgs,
   ClientLoaderFunction,
   ClientLoaderFunctionArgs,
+  HeadersArgs,
+  HeadersFunction,
   MetaArgs,
   MetaDescriptor,
   MetaFunction,
   LinksFunction,
+  ServerRouteModule,
```

Comment:
This might be weird, and potentially even a sign that this doesn't belong in type docs?

This is not a user-facing type but the only way to get the typedoc generated is to make it public API so we have to export it even though we never want anyone to import it and use it.